kong/api/routes/cluster_status.lua

```diff
@@ -1,18 +1,19 @@
-local clustering = require("kong.clustering")
-local kong = kong
+local endpoints = require "kong.api.endpoints"
 
 
 return {
-  ["/clustering/status"] = {
-    GET = function(self, db, helpers)
+  ["/cluster_status"] = {
```

Comment:
We should do some forward thinking around naming and user-facing API here, some raw thoughts:
-  This endpoint returns is  a list of nodes connected and not the health of the overall cluster. We should reconsider the name of this endpoint. 
- `/cluster` could be a namespace under which we can introduce more endpoints in future
- `/cluster/nodes` or `/cluster/nodes/proxy` should be the name of this endpoint
- In future, `/cluster/nodes?type=control` could also provide all the control plane nodes that are part of the cluster 
- In future, the same endpoint can be potentially introduced in the "traditional" mode to provide easy observability of how many kong nodes are connected to the same db

Reply:
Also please note that the API will need a DELETE endpoint at some point to prune the nodes that are dead in the cluster. While most users will be okay with regular automated pruning done by control plane nodes but I expect the above to be also come up.

Reply:
@hbagdi Yes, we will probably add these (including auto-deletion of unneeded rows) after the alpha.